packages/entity-list/src/components/SearchFilterList/SearchFilterList.js

```diff
@@ -1,33 +1,71 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import {Typography, Button} from 'tocco-ui'
+import {Typography, Button, theme, StyledSpan} from 'tocco-ui'
 import styled from 'styled-components'
 
+const SearchFilterWrapper = styled.div`
+  margin-top: .4rem;
+  margin-bottom: .4rem;
+`
+
+const StyledSpanSearchFilter = styled(StyledSpan)`
```

Comment:
Is it not possible to make a Typography.Span selector in StyledSearchFilterButton and set display and width there?
(is the Span needed anyway?)